### ISSUE TITLE
COMPAT: bottleneck 1.2.0

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas#14495]
just need to verify things, should hopefully be ok.

https://pypi.python.org/pypi/Bottleneck

its not up on conda-forge (or defaults ATM). so need to watch for that and test.

**Comment by jreback** [comment]
once this PR by @wesm is merged [here](https://github.com/conda-forge/bottleneck-feedstock/pull/7), this should be updated. We don't utilize conda-forge channel much though (just for a single matplotlib install now), but will test adding it as an auxillary channel with lower priority.
